scripts/conversionScripts/conversion_from_old_hdf5_to_new/conversionFromOldToNewDatabase.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os, sys, json
+
+#find TreeStructure module
+sys.path.append(os.path.normpath(os.path.join(os.path.dirname(__file__),os.pardir,os.pardir,os.pardir,'framework')))
+
+from reader_old_HDF5_format import OldHDF5Database
+from h5py_interface_creator import hdf5Database
+
+import MessageHandler
+
+mh = MessageHandler.MessageHandler()
+mh.initialize({'verbosity':'debug', 'callerLength':10, 'tagLength':10})
```

Comment:
Did you need a message handler for something? I don't see it used below.

Reply:
yes...because I am using the ```h5py_interface_creator.py``` that is using it (I am passing it in the constructor)